### DISCUSSION TITLE
Editing Personal Workspace as "read only" User

### DISCUSSION THREAD

**Discussion by honestegg** [datalust/seq-tickets#1932] [Help] [answered]
As a "read only" User, should you be able to edit your own "Personal" Workspace? How about saving personal signals? It seems like you can't and I want to make sure that behavior is expected. 

Seq version: 2023.3.9691

**Comment by nblumhardt** [comment, marked as answer]
Thanks for checking in! Currently no, read-only users are truly read-only and can only change a handful of preferences.

See also: https://github.com/datalust/seq-tickets/issues/1817

**Reply by honestegg** [reply]
It would be nice to have a role that at least allowed a user to edit their own Personal workspace

**Reply by nblumhardt** [reply]
Agreed; I think the right way for us to enable this will be to split the current `Read` and `Write` permissions down into finer-grained ones (the current permissions are very coarse). Will take another look at this when there's a spare moment 👍
